content/docs/command-reference/push.md

```diff
@@ -155,37 +155,51 @@ a [pipeline](/doc/command-reference/pipeline) has been setup with these
 [stages](/doc/command-reference/run):
 
 ```dvc
-$ dvc pipeline show
-data/Posts.xml.zip.dvc
-Posts.xml.dvc
-Posts.tsv.dvc
-Posts-test.tsv.dvc
-matrix-train.p.dvc
-model.p.dvc
-Dvcfile
+$ dvc dag
+         +---------+
+         | prepare |
+         +---------+
+              *
+              *
+              *
+        +-----------+
+        | featurize |
+        +-----------+
+         **        **
+       **            *
+      *               **
++-------+               *
+| train |             **
++-------+            *
+         **        **
+           **    **
+             *  *
+        +----------+
+        | evaluate |
+        +----------+
 ```
 
 Imagine the <abbr>projects</abbr> has been modified such that the
 <abbr>outputs</abbr> of some of these stages need to be uploaded to
 [remote storage](/doc/command-reference/remote).
 
 ```dvc
-$ dvc status --cloud
-
-  new:            data/model.p
-  new:            data/matrix-test.p
-  new:            data/matrix-train.p
+$ dvc status -c
+    new:            data/featurize/train.pkl
+    new:            data/featurize/train.pkl
+    new:            data/prepared/train.tsv
+    new:            data/prepared/test.tsv
```

Comment:
@jorgeorpinel `status`, `pull`, `push` doesn't seem to work for example-get-started repo. I have made up this output myself.

Reply:
Should work. What errors are you getting?

But let's not change the example stages. You can make dummy data files and stages based on the existing file and stage names in this example e.g. `Posts.xml`, `data/model.p, etc.

Reply:
I can think the problem is with the remote of `example-get-started` repo . This was the error:
`ERROR: unexpected error - Unable to locate credentials`

Reply:
Strange. At `dvc status -c`?

Reply:
Anyway, is this even relevant anymore? 🙂